pkg/manifest/common.go

```diff
@@ -0,0 +1,22 @@
+package manifest
+
+import (
+	"github.com/osbuild/images/pkg/disk"
+	"github.com/osbuild/images/pkg/osbuild"
+)
+
+// filesystemConfigStages generates either an org.osbuild.fstab stage or a
+// collection of org.osbuild.systemd.unit.create stages for .mount and .swap
+// units (and an org.osbuild.systemd stage to enable them) depending on the
+// pipeline configuration.
+func filesystemConfigStages(pt *disk.PartitionTable, mountUnits bool) ([]*osbuild.Stage, error) {
```

Comment:
Perhaps it could make sense to pass in an interface here instead that this function can use to determine all it needs to create any filesystem configs. There's no need for it *now* so feel free to ignore.